File: Java/PostMortem/src/com/nastajus/Fizzbuzz.java
package com.nastajus;

/**
 * Created by IAN on 24/11/14.
 */
public class Fizzbuzz {
    public static void run(){

        //modulus negatives in Java
        for(int i=-50; i<=50; i++){
            System.out.print(" ");

            if ( i % 3 == 0 ){
                System.out.print("fizz");
            }
            if ( i % 5 == 0 ){
                System.out.print("buzz");
            }

            if ( i % 3 != 0 && i % 5 != 0){
                System.out.print(i);
            }

            if ( i % 30 == 0){
                System.out.println("");
            }
        }

    }
}

File: C#/PostMortem/PostMortem/Fizzbuzz.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PostMortem
{
    class Fizzbuzz
    {
        //modulus negatives in Java
        public static void run(){
            for (int i = -50; i <= 50; i++)
            {
                Console.Write(" ");

                if (i % 3 == 0)
                {
                    Console.Write("fizz");
                }
                if (i % 5 == 0)
                {
                    Console.Write("buzz");
                }

                if (i % 3 != 0 && i % 5 != 0)
                {
                    Console.Write(i);
                }

                //if (i % 30 == 0)
                //{
                //    Console.WriteLine("");
                //}
            }
        }
    }
}
